autokeras/auto_model.py

```diff
@@ -42,16 +44,17 @@ def __init__(self,
                  max_trials=100,
                  directory=None,
                  objective='val_loss',
+                 tuner='greedy',
                  seed=None):
         self.inputs = nest.flatten(inputs)
         self.outputs = nest.flatten(outputs)
         self.name = name
-        self.tuner = None
         self.max_trials = max_trials
         self.directory = directory
         self.seed = seed
         self.hyper_graph = None
         self.objective = objective
+        self.tuner = tuner
```

Comment:
Why not `self.tuner = tuner_module.get_tuner_class(self.tuner)`? This would be more consistent.